style.css

```diff
@@ -0,0 +1,6 @@
+header {
+    position: fixed;
+    width: 100%;
+    top: 0;
```

Comment:
Kindly add two white space before each line of code. it will look good indentation